### ISSUE TITLE
Clarify line between Mozilla and Network

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#73]
@HPaulJohnson @KevZawacki – The site tells three interwoven stories about the Network, Mozilla, and a healthy internet.  Based on staff discussions, we might get more explicit: What parts serve/promote the network? What parts promote Mozilla?

 - **Programs > Mozilla ** – I wonder if it's useful to reframe intro text on Program page to distinguish programs from Projects and People. Something like: This is how Mozilla fuels/catapults the network.  

 - **Projects > Network ** – Projects should promote work by network members. We should remove Mozilla items from the homepage and show them sparingly on the Projects page. See discussion and options: https://github.com/mozilla/network/issues/70

 - **People > Network ** – People page should emphasize net members. Staff should be sprinkled in, because they are awesome, but nobody should confuse this for a staff listing.

cc @taisdesouzalessa  @sabrinang

**Comment by xmatthewx** [comment]
tl;dr – @HPaulJohnson @KevZawacki – should we craft a new phrase for program page that describes these specifically as Mozilla things that support/fuel the network?

**Comment by KevZawacki** [comment]
@xmatthewx Yes, we can certainly tweak it to be more explicit. How about (new text in bold):

[Hed] Fellowships, Research and More
[Dek] Fellowships empowering civic technologists. Laboratories studying emerging tech. Grants supporting web literacy. **Mozilla's deep and far-reaching programs are built to support the network**.

And throughout the rest of the Program page content, change "we" to "Mozilla" for additional clarity.

**Comment by xmatthewx** [comment]
Copy is good. We'll need to see it in a mockup whether it stands out enough. I'm wondering if this page needs a different treatment, a color switch to signal the change. 

What about even more explicit?
[Hed] Mozilla supports the network
 ...or...
[Hed] Mozilla's programs
[Dek] Fellowships empowering civic technologists. Laboratories studying emerging tech. Grants supporting web literacy. Mozilla programs for the network.

**Comment by KevZawacki** [comment]
@xmatthewx That works! I like your last treatment:

[Hed] Mozilla's programs
[Dek] Fellowships empowering civic technologists. Laboratories studying emerging tech. Grants supporting web literacy. Mozilla programs for the network.

**Comment by hannahkane** [comment]
@xmatthewx - wondering if we can close this ticket. I believe we've addressed these issues in various ways.

**Comment by xmatthewx** [comment]
closing. addressed elsewhere.
